frontend/src/components/commons/BackToTop.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import scrollToTop from "../../helpers/scrollToTop";
+
+const styles = {
+  hiddenButton: {
+    visibility: "hidden"
+  },
+  displayedButton: {
+    display: "block",
+    position: "fixed",
+    bottom: "5%",
+    right: "3%"
+  }
+};
+
+class BackToTop extends Component {
+  state = {
+    displayButton: "hidden"
```

Comment:
It'd be better to define an enum instead of using strings here: https://github.com/code-for-canada/project-thundercat/blob/master/frontend/src/components/eMIB/Emib.jsx#L8

Reply:
I'm now using true and false as state.
See commit #https://github.com/code-for-canada/project-thundercat/pull/87/commits/f9dd094e5d83a03971d4050f0b95ed4036af1101